sdk/couchbase-core/src/configwatcher.rs

```diff
@@ -144,18 +144,21 @@ where
     }
 
     async fn poll_one(&self, endpoint: String) -> Result<ParsedConfig> {
+        debug!("Polling config from {}", &endpoint);
+
         let client = self.kv_client_manager.get_client(endpoint).await?;
 
-        let addr = client.remote_addr();
-        let host = addr.ip();
+        let hostname = client.remote_hostname();
 
         let resp = client
             .get_cluster_config(GetClusterConfigRequest {})
             .await?;
 
-        let config: TerseConfig = serde_json::from_slice(resp.config.as_slice())?;
+        let config = parse_terse_config(&resp.config, hostname)?;
```

Comment:
This is a bit confusingly named, considering the call of another function of the same name (albeit a different namespace) immediately after

Reply:
I've added the module name to the call to try to make it more clear. IMO it is the correct name with regards to the module itself though.